tests/e2e/helper_test.go

```diff
@@ -436,6 +437,7 @@ func ensureServicemeshOperators(t *testing.T, tc *testContext) error { //nolint:
 	ops := []string{
 		serverlessOpName,
 		servicemeshOpName,
+		authorinoOpName,
```

Comment:
I created that time bomb, but probably makes sense to rename ensureServicemeshOperators to ensureDependencyOperators or something (did @VaishnaviHire already do it somewhere?)

Reply:
updated the fuction name to `ensureDependencyOperators`